fcrepo-search-impl/src/main/java/org/fcrepo/search/impl/DbSearchIndexImpl.java

```diff
@@ -188,21 +199,26 @@ public void addUpdateIndex(final ResourceHeaders resourceHeaders) {
             try {
 
                 final var params = new MapSqlParameterSource();
-                params.addValue( fedoraIdParam, fullId);
+                params.addValue(fedoraIdParam, fullId);
                 final var modifiedParam = "modified";
+                final var contentSizeParam = "content_size";
+                final var mimetypeParam = "mime_type";
                 params.addValue(modifiedParam, new Timestamp(resourceHeaders.getLastModifiedDate().toEpochMilli()));
+                params.addValue(mimetypeParam, resourceHeaders.getMimeType());
```

Comment:
I noticed while reviewing this code that you're adding timestamps using `Timestamp` but querying using `Date`. Should they both use the same type?

Reply:
I will take a look.

Reply:
@pwinckles :  where do you see the querying using date?

Reply:
oh - forget it.  I see.